CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Labels
 * Cromwell has removed most of the formatting restrictions from custom labels. Please check the [README](README.md#label-format) for more detailed documentation.
 * Custom labels won't be submitted to Google backend as they are now decoupled from Google's default labels.
+* Cromwell now publishes the labels as soon as the workflow is submitted (whether started or on hold). If the labels are invalid, workflow will not be submitted and request will fail.
```

Comment:
Maybe "the workflow will not be submitted"?

Reply:
oh no, i meant to add the "the". I think. You shouldn't take grammar advice from me. That's @mcovarr 's bag